src/lambdas/tenderly-simulate.ts

```diff
@@ -0,0 +1,47 @@
+import fetch from 'isomorphic-fetch';
+
+const { TENDERLY_USER, TENDERLY_PROJECT, TENDERLY_ACCESS_KEY } = process.env;
+
+const TENDERLY_ENDPOINT = `https://api.tenderly.co/api/v1/account/${TENDERLY_USER}/project/${TENDERLY_PROJECT}/simulate`;
+
+function formatResponse(statusCode, body) {
+  return {
+    statusCode,
+    headers: {
+      'Content-Type': 'application/json',
+      'Access-Control-Allow-Headers': 'Content-Type',
+      'Access-Control-Allow-Origin': '*',
+      'Access-Control-Allow-Methods': 'OPTIONS,POST',
+    },
+    body,
+  };
+}
+
+export const handler = async ({ body, ...event }: any = {}): Promise<any> => {
+  if (!body) {
+    return formatResponse(
+      400,
+      'Error: invalid request, you are missing the request body'
+    );
+  }
+  console.log({ body: body, event });
+  try {
+    const res = await fetch(TENDERLY_ENDPOINT, {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+        'X-Access-Key': Buffer.from(
+          `${TENDERLY_ACCESS_KEY}:${TENDERLY_ACCESS_KEY}`
+        ).toString('base64'),
+      },
+      body: typeof body == 'object' ? body : JSON.parse(body),
```

Comment:
Is there any security risks if we're passing arbitrary user data into our tenderly account?

Reply:
cc @markusbkoch

Reply:
I don't think so, but @mikemcdonald has a better sense for these things

Reply:
Normally this logic would raise some red flags, but looking at the simulate API docs there is only a single endpoint request for the simulation and no endpoints around account controls, changing settings, etc. That being said, it should be very easy to validate the body JSON before the request is sent to ensure it only has the keys available for the simulation.

Reply:
Thanks for the feedback. I added a check of the fields in the request body